pkg/scheduler/framework/runtime/framework.go

```diff
@@ -975,7 +977,9 @@ func (f *frameworkImpl) RunPermitPlugins(ctx context.Context, state *framework.C
 			if status.IsUnschedulable() {
 				msg := fmt.Sprintf("rejected pod %q by permit plugin %q: %v", pod.Name, pl.Name(), status.Message())
 				klog.V(4).Infof(msg)
-				return framework.NewStatus(status.Code(), msg)
+				s := framework.NewStatus(status.Code(), msg)
```

Comment:
ditto, we should append the reasons. I think this was implemented before we allowed multiple reasons in status.